src/components/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -87,102 +81,143 @@ const EmployeeClaimWrapper = styled('div')({
-                }}
-              >
-                <MuiMenuItem key={'True'} value={'True'}>
-                  True
-                </MuiMenuItem>
-                <MuiMenuItem key={'False'} value={'False'}>
-                  False
+  contract,
+}) => {
+  const [contracts] = useContracts(memberId)
+  const [setContractForClaim] = useSetContractForClaim()
+  const [setCoveringEmployee] = useSetCoveringEmployee()
+  const [updateClaimState] = useUpdateClaimState()
+
+  return (
+    <>
+      <Paper>
+        <h3>Claim Information</h3>
+        {recordingUrl && (
+          <AudioWrapper>
+            <p>
+              Registered at:{' '}
+              {format(parseISO(registrationDate), 'yyyy-MM-dd hh:mm:ss')}
+            </p>
+            <Audio controls>
+              <source src={recordingUrl} type="audio/aac" />
+            </Audio>
+            <Audio controls>
+              <source
+                src={getUrlWithoutParameters(recordingUrl)}
+                type="audio/aac"
+              />
+            </Audio>
+            <DonloadClaimFile
+              href={recordingUrl}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Download claim file
+            </DonloadClaimFile>
+            <DonloadClaimFile
+              href={getUrlWithoutParameters(recordingUrl)}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Download claim file
+            </DonloadClaimFile>
+          </AudioWrapper>
+        )}
+        <StatusWrapper>
+          <MuiInputLabel shrink>Status</MuiInputLabel>
+          <MuiSelect
+            value={state}
+            onChange={async (event) => {
+              await updateClaimState(
+                updateClaimStateOptions(claimId, validateSelectOption(event)),
+              )
+              await refetchPage()
+            }}
+          >
+            {Object.values(ClaimState).map((s) => (
+              <MuiMenuItem key={s} value={s}>
+                {s}
+              </MuiMenuItem>
+            ))}
+          </MuiSelect>
+        </StatusWrapper>
+        <EmployeeClaimWrapper>
+          <MuiInputLabel shrink>Employee Claim</MuiInputLabel>
+          <MuiSelect
+            value={coveringEmployee ? 'True' : 'False'}
+            onChange={async (event) => {
+              await setCoveringEmployee(
+                setCoveringEmployeeOptions(
+                  claimId,
+                  validateSelectEmployeeClaimOption(event),
+                ),
+              )
+              await sleep(1000)
+              await refetchPage()
+            }}
+          >
+            <MuiMenuItem key={'True'} value={'True'}>
+              True
+            </MuiMenuItem>
+            <MuiMenuItem key={'False'} value={'False'}>
+              False
+            </MuiMenuItem>
+          </MuiSelect>
+        </EmployeeClaimWrapper>
+        {contracts && (
+          <EmployeeClaimWrapper>
```

Comment:
We should probably rename this wrapper
```suggestion
          <SelectWrapper>
```